src/demo-app/a11y/progress-spinner/progress-spinner-a11y.html

```diff
@@ -0,0 +1,13 @@
+<section>
+  <h2>Loading indicator (Indeterminate progress spinner)</h2>
+  <md-spinner color="indeterminate" [strokeWidth]="1"></md-spinner>
```

Comment:
Also just occurred to me that each progress component should have an `aria-label`